source/manual/removing-a-user-from-puppet.html.md

```diff
@@ -19,14 +19,14 @@ will remain on our servers forever more.
    [example][absent-example].
 3. Once this has been raised as a PR and merged, deploy Puppet to all
    environments.
-4. Create another PR for Puppet that:
+4. Create a PR in [GOV.UK secrets][govuk-secrets] that:
+  - Removes the user from [production hieradata][production-hieradata]. Read [what to do when someone leaves][what-to-do-when-someone-leaves]
+  - Removes the user from [AWS production hieradata][aws-production-hieradata]. Read [what to do when someone leaves][what-to-do-when-someone-leaves]
+5. Create another PR for Puppet that:
```

Comment:
Can we use the Markdown `1.` trick for these numbers?

Reply:
yeah - let's have a go 2cc6bfe